packages/grid/x-grid/src/tests/columns.XGrid.test.tsx

```diff
@@ -164,5 +164,94 @@ describe('<XGrid /> - Columns', () => {
       expect(onColumnWidthChange.callCount).to.equal(1);
       expect(onColumnWidthChange.args[0][0].width).to.equal(120);
     });
+
+    describe('flex resizing', () => {
+      before(function beforeHook() {
+        if (isJSDOM) {
+          // Need layouting
+          this.skip();
+        }
+      });
+
+      it('should resize the flex width after resizing a column with api', () => {
+        const twoColumns = [
+          { field: 'id', width: 100, flex: 1 },
+          { field: 'brand', width: 100 },
+        ];
+
+        render(<Test columns={twoColumns} />);
+
+        expect(getColumnHeaderCell(0).style.width).to.equal('198px');
```

Comment:
We have utils for this, the main advantage is that it throw if the component can't be found and render the element details when the assertion fails. I propose we use it here, x the other cases
```suggestion
        expect(getColumnHeaderCell(0)).toHaveInlineStyle({ width: '198px' });
```